kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -1,117 +1,108 @@
+
+The script assumes either the image container-registry.oracle.com/database/enterprise:12.2.0.1-slim is available in docker image repository or an ImagePullSecret is created for container-registry.oracle.com. See the script create-image-pull-secret.sh to create secret on container-registry.oracle.com .
+
+$ ./start-db-service.sh -h    
+usage: ./start-db-service.sh -p <nodeport> -i <image> -s <docker-store> [-h]
+ -i  Oracle DB Image (optional)
+    (default: container-registry.oracle.com/database/enterprise:12.2.0.1-slim ) 
+  -p DB Service NodePort (optional)
+    (default: 30011) 
+  -s DB Image PullSecret  (optional)
+    (default: docker-store) 
+  -h Help
+
+$ ./start-db-service.sh     
+NodePort[30011] ImagePullSecret[docker-store] Image[container-registry.oracle.com/database/enterprise:12.2.0.1-slim]
 deployment.extensions/oracle-db created
 service/oracle-db created
-[oracle-db-756f9b99fd-r6ghb] already initialized .. 
+[oracle-db-54667dfd5f-76sxf] already initialized .. 
 Checking Pod READY column for State [1/1]
+Pod [oracle-db-54667dfd5f-76sxf] Status is Ready Iter [1/60]
 NAME                         READY   STATUS    RESTARTS   AGE
-oracle-db-756f9b99fd-r6ghb   1/1     Running   0          3s
-NAME                         READY   STATUS    RESTARTS   AGE
-oracle-db-756f9b99fd-r6ghb   1/1     Running   0          4s
-NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
-kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP          16d
-oracle-db    NodePort    10.100.148.17   <none>        1521:30011/TCP   3s
-Oracle DB service is RUNNING with external NodePort [30011]
-
-```
-The parameters are as follows:
-
-```  
-  -p external NodePort for the Service (default is 30011) 
-```
-```
-The script create a Oracle DB Service in default Namesapce with default Credential that comes with the Oracle Database Slim image.
-$ kubectl get po
+oracle-db-54667dfd5f-76sxf   1/1     Running   0          8s
 NAME                         READY   STATUS    RESTARTS   AGE
-oracle-db-756f9b99fd-ch7xt   1/1     Running   0          33m
-$ kubectl get svc
+oracle-db-54667dfd5f-76sxf   1/1     Running   0          8s
 NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
-oracle-db    NodePort    10.99.94.157   <none>        1521:30011/TCP   32m
+kubernetes   ClusterIP   10.96.0.1      <none>        443/TCP          27d
+oracle-db    NodePort    10.99.58.137   <none>        1521:30011/TCP   9s
+Oracle DB service is RUNNING with NodePort [30011]
 
-The DB Connection String is oracle-db.default.svc.cluster.local:1521/devpdb.k8s which can be used as  rcuDatabaseURL parameter to domain.input.yaml file while creating a Fusion Middleware domain in Operator Environment 
+The DB Connection String is oracle-db.default.svc.cluster.local:1521/devpdb.k8s which can be used as  rcuDatabaseURL parameter to domain.input.yaml file while creating a Fusion Middleware domain.
```

Comment:
Backticks around oracle-db.default.svc.cluster.local:1521/devpdb.k8s and rcuDatabaseURL.